internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -76,6 +84,46 @@ func (handler StorageHandler) addContext(next func(http.ResponseWriter, *http.Re
 	})
 }
 
+// processGetAsyncRequest is used to handle Async Get Requests
+func (handler StorageHandler) processGetAsyncRequest(writer http.ResponseWriter, request *http.Request) {
+	vars := mux.Vars(request)
+	deviceName := vars[deviceNameKey]
+	resourceName := vars[resourceNameKey]
+
+	handler.logger.Debug(fmt.Sprintf("Received POST for Device=%s Resource=%s", deviceName, resourceName))
+
+	_, err := handler.service.GetDeviceByName(deviceName)
+	if err != nil {
+		handler.logger.Error(fmt.Sprintf("Incoming reading ignored. Device '%s' not found", deviceName))
+		http.Error(writer, fmt.Sprintf("Device not found"), http.StatusNotFound)
+		return
+	}
+	_, ok := handler.service.DeviceResource(deviceName, resourceName, "get")
```

Comment:
I did that in accordance to 
https://github.com/lf-edge/edge-home-orchestration-go/blob/e9e2278d04d31c0cc1c8daec6dac840788c6bcd7/internal/controller/storagemgr/storagedriver/storagehandler.go#L141 

In case err would be better, I'll change. Please suggest.

Reply:
Make sense. use `ok`.